.ci/install_qemu_lite.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+#
+# Copyright (c) 2017-2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set -e
+
+arch=$(arch)
+
+if [ "$#" -ne 3 ]; then
+    echo "Usage: $0 <CLEAR_RELEASE> <QEMU_LITE_VERSION> <DISTRO>"
+    echo "       Install the QEMU_LITE_VERSION from clear CLEAR_RELEASE."
+    exit 1
+fi
+
+clear_release="$1"
+qemu_lite_version="$2"
+distro="$3"
+qemu_lite_bin="qemu-lite-bin-${qemu_lite_version}.${arch}.rpm"
+qemu_lite_data="qemu-lite-data-${qemu_lite_version}.${arch}.rpm"
+
+echo -e "Install qemu-lite ${qemu_lite_version}"
+
+# download packages
+curl -LO "https://download.clearlinux.org/releases/${clear_release}/clear/${arch}/os/Packages/${qemu_lite_bin}"
```

Comment:
At the risk of stating the obvious, because we're re-using CLR binaries, this is only going to work for x86_64. As such, I'd add a test below the `arch=$(arch)` line to die if any other architecture is detected for the time being.

/cc @bergwolf.